module/os/linux/spl/spl-kmem.c

```diff
@@ -131,6 +131,27 @@ kmem_strfree(char *str)
 }
 EXPORT_SYMBOL(kmem_strfree);
 
+int
+kmem_scnprintf(char *restrict str, size_t size, const char *restrict fmt, ...)
```

Comment:
Note to self: Use a preprocessor definition to remap this to scnprintf in the Linux kernel.